llama_index/indices/document_summary/retrievers.py

```diff
@@ -144,21 +144,10 @@ def _get_embeddings(
         """Get top nodes by similarity to the query."""
         embed_model = self._index.service_context.embed_model
         if query_bundle.embedding is None:
-            event_id = self._index._service_context.callback_manager.on_event_start(
-                CBEventType.EMBEDDING
-            )
             query_bundle.embedding = embed_model.get_agg_embedding_from_queries(
                 query_bundle.embedding_strs
             )
-            self._index._service_context.callback_manager.on_event_end(
-                CBEventType.EMBEDDING,
-                payload={EventPayload.CHUNKS: query_bundle.embedding_strs},
-                event_id=event_id,
-            )
 
-        event_id = self._index._service_context.callback_manager.on_event_start(
-            CBEventType.EMBEDDING
-        )
```

Comment:
Is nixing this intentional? This could have consequences on other callbacks potentially? or is the idea that an embedding callback only gets called if there are embedding(s)? Just confirming - maybe take a look at the git blame here.

Reply:
These are already being logged by the underlying embed model, no need to log them twice

Reply:
Ah god it, 🙏